### PR TITLE
Init gpt.js in AudienceActionLocalFlow.

### PR DESCRIPTION
**Pull request by mhkawano**
This inits gpt.js so that we can begin showing ads. I also install gpt.js types into the project. Same as the last PR, this should not affect existing behavior, as nothing branches into it. The actual UI is not built yet. The next PR will probably introduce a simple UI, and the one after that will introduce a flag and allow the autopromptmanager to call this.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "@babel/core": "7.22.9",
     "@babel/eslint-parser": "7.22.9",
     "@babel/preset-env": "7.22.9",
+    "@types/google-publisher-tag": "^1.20230724.0",
```
[the remark this answers is not in the document]
Comment (mhkawano) [reply package.json]: Done.

[the remark this answers is not in the document]
Comment (mhkawano) [reply package.json]: Ah, guess not. It seems like it can't build if put in optional.

[the remark this answers is not in the document]
Comment (ChrisAntaki) [reply package.json:37]: Sorry, that makes sense. By the way, could you remove the `^` from the version #?

[the remark this answers is not in the document]
Comment (mhkawano) [reply package.json]: Done.

**Pull request merged**